src/middlewared/middlewared/etc_files/generate_ssl_certs.py

```diff
@@ -19,9 +20,24 @@ def write_certificates(certs):
             with open(cert['csr_path'], 'w') as f:
                 f.write(cert['CSR'])
 
+    """
+    Write unified CA certificate file for use with LDAP.
+    """
+    if not cacerts:
+        shutil.copyfile('/usr/local/share/certs/ca-root-nss.crt',
+                        '/etc/ssl/truenas_cacerts.pem')
+    else:
+        with open('/etc/ssl/truenas_cacerts.pem', 'a+') as f:
```

Comment:
Is append still intended here? I suspect it should be truncating the file instead.

Reply:
No, it is no longer intended.